src/stdlib/lib.c

```diff
@@ -196,6 +196,21 @@ value_p printd(value_p whatever, value_p text) {
 	return result;
 }
 
+value_p to_string(value_p val) {
+		if(assertSingleNumber(val)) {
+			double possible_num = val->numericVal;
+			int size = snprintf(NULL, 0, "%f", possible_num);
+			char *converted_str = malloc(size + 1);
+			sprintf(converted_str, "%f", possible_num);
+			value_p result = box_value_string(new_string(converted_str));
+			return result;
+		}
+		else if(assertSingleString(val)) return val;
+
+		// If the struct does not hold a string or number, return empty?
```

Comment:
I'm not sure if we have a defined behavior for subranges.
I assume it should be whatever behavior is described in test_access_selected_range_2.xtnd and such.